test/gurka/gurka.h

```diff
@@ -163,10 +163,11 @@ build_config(const std::string& tiledir,
   return ptree;
 }
 
-std::string build_valhalla_route_request(const map& map,
-                                         const std::vector<std::string>& waypoints,
-                                         const std::string& costing = "auto",
-                                         const std::string& datetime = "") {
+std::string
+build_valhalla_route_request(const map& map,
+                             const std::vector<std::string>& waypoints,
+                             const std::string& costing = "auto",
+                             const std::unordered_map<std::string, std::string>& options = {}) {
```

Comment:
we needed somethign more flexible to be able to easily add options to the requests (alternatively you can make your own request entirely but this was too tempting)